demos/embeddings/stellargraph-keras-node2vec.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<blockquote>An example of implementing the keras Node2Vec representation learning algorithm using components from the stellargraph library</blockquote><p>How about something like : "This example demonstrates how to apply components from the stellargraph library to perform representation learning via Node2Vec. This uses a keras implementation of Node2Vec available in stellargraph instead of the reference implementation provided by <code>gensim</code>."? Or I think we could at least replace "implementing" with "demonstrating".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thanks Kevin! I have change the introduction according to your suggestion and add an extra description "This implementation provides flexible interfaces to downstream tasks for end-to-end learning."